microsite/data/on-demand/20211117-01.yaml

```diff
@@ -0,0 +1,7 @@
+---
+title: Community Sessions
+date: November 17, 2021
+category: Meetup
+description: At this month’s adopters session, get the scoop on Spotify’s plan for paid plugins, a first look at Box’s DevPortal, and answers to big questions, like “How will I know if Backstage will work at a large company like mine?”
```

Comment:
Does your locale have some odd quotation mark setting? :) I know it's a huge nitpick but it caught my eye that they looked odd